raiden/transfer/state.py

```diff
@@ -23,6 +25,231 @@ def balanceproof_from_envelope(envelope_message):
     )
 
 
+class NodeState(State):
+    """ Umbrela object that stores all the node state.
+    For each registry smart contract there must be a payment network, withn the
+    payment network the existing token networks and channels are registered.
+    """
+
+    __slots__ = (
+        'block_number',
+        'identifiers_to_paymentnetworks',
+        'nodeaddresses_to_networkstates',
+        'payment_mapping',
+    )
+
+    def __init__(self, block_number: typing.block_number):
+        if not isinstance(block_number, typing.block_number):
+            raise ValueError('block_number must be an block_number')
+
+        self.block_number = block_number
+        self.identifiers_to_paymentnetworks = dict()
+        self.nodeaddresses_to_networkstates = dict()
+        self.payment_mapping = PaymentMappingState()
+
+    def __repr__(self):
+        return '<NodeState block:{} networks:{} qtd_transfers:{}>'.format(
+            self.block_number,
+            lpex(self.identifiers_to_paymentnetworks.keys()),
+            len(self.payment_mapping.hashlocks_to_task),
+        )
+
+    def __eq__(self, other):
+        return (
+            isinstance(other, NodeState) and
+            self.block_number == other.block_number and
+            self.identifiers_to_paymentnetworks == other.identifiers_to_paymentnetworks and
+            self.nodeaddresses_to_networkstates == other.nodeaddresses_to_networkstates and
+            self.payment_mapping == other.payment_mapping
+        )
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+
+class PaymentNetworkState(State):
+    """ Corresponds to a registry smart contract. """
+
+    __slots__ = (
+        'address',
+        'tokensidentifiers_to_tokennetworks',
+        'tokenaddresses_to_tokennetworks',
+    )
+
+    def __init__(
+            self,
+            address: typing.address,
+            token_network_list: typing.List['TokenNetworkState']):
+
+        if not isinstance(address, typing.address):
+            raise ValueError('address must be an address instance')
+
+        self.address = address
+        self.tokensidentifiers_to_tokennetworks = {
```

Comment:
Should that be `tokenidentifiers`, so without the 's' in the middle?

Reply:
it is already. Did you change that yourself @palango ?

Reply:
nope

Reply:
But I still see the s in the middle

Reply:
aaah lol after reading it 3 times I now understand what you meant

Reply:
haha, I have to work on my writing then ;)